ansible/vars/icds/icds_public.yml

```diff
@@ -0,0 +1,195 @@
+SITE_HOST: 'icds-cas.gov.in'
+NO_WWW_SITE_HOST: 'icds-cas.gov.in'
+deploy_env: icds
+
+riakcs_s3_access_key: "{{ secrets.RIAKCS_S3_ACCESS_KEY }}"
+riakcs_s3_secret_key: "{{ secrets.RIAKCS_S3_SECRET_KEY }}"
+
+active_sites:
+  cchq_ssl: True
```

Comment:
this is a good question. for our DNS I think we can move the `cas.commcarehq.org` cert over to this server. for NIC's DNS we would need a new cert. we can add this to list of things we are requesting for Monday